UI overhaul

the-dunny opened the pull request:
After looking at his example for project 1... I started thinking that UI should absolutely be singleton pattern.  Here is what I did.

dailydev-tech committed:
testing to see where this commit and push goes.  hopefully just to my
fork

dailydev-tech committed:
I changed the user interface so that it is a singleton pattern design.
Used do-while to keep program running after non-int input

dailydev-tech committed:
Used constants instead of hard-coding

jackhenry reviewed (commented):
Good idea using constants.

jackhenry commented on src/UserInterface.java:
```diff
@@ -0,0 +1,191 @@
+
+public class UserInterface {
+	private static UserInterface userInterface;
+
```
a blank line here is against professor's style guide.

jackhenry commented on src/UserInterface.java:
```diff
@@ -0,0 +1,191 @@
+				String line = reader.readLine();
+				String[] words = line.split("\\s"); 
+				return words[0];
+
```
Extra blank line.

jackhenry reviewed (changes requested):
Check spacing. The style guide wants compact code.

jackhenry committed:
Delete Driver2.java

Commit:
deleted blank spaces and set up Driver class.  changed some methods in
UserInterface to private that didnt need to be public.  changed showUI()
to showUserInterface()

Commit:
missed a blank line.

jackhenry reviewed (approved):
Looks good to me.
